### PR TITLE
Create seeder for colors

### PR DESCRIPTION
**Pull request by lanorrron**
Se creó un seeder para la tabla colors 
Se agregaron los siguientes colores por defecto:

- Rojo
- Negro
- Amarillo
- Blanco

### CODE REVIEW

File: database/seeders/ColorSeeder.php [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+     *
+     * @return void
+     */
+    public function run()
+    {
+        DB::table('colors')->insert([
+           ['name'=>'Rojo'],
+           ['name'=>'Negro'],
+           ['name'=>'Blanco'],
+           ['name'=>'Amarillo'],
```
Comment (ttimbal) [inline database/seeders/ColorSeeder.php:22]: why use DB and not the direct model?

File: database/seeders/DatabaseSeeder.php [hunk 2]
```diff
@@ -15,7 +15,9 @@ class DatabaseSeeder extends Seeder
     public function run()
     {
     $this->call([
-       SizeSeeder::class
+       SizeSeeder::class,
+        ColorSeeder::class
```
Comment (ttimbal) [inline database/seeders/DatabaseSeeder.php:19]: Remove the space to maintain all at the same style

**Review by ttimbal — CHANGES_REQUESTED** [review]

File: database/seeders/ColorSeeder.php (hunk 1 above)
Comment (lanorrron) [reply database/seeders/ColorSeeder.php:22]: a que se refiere?

Comment (ttimbal) [reply database/seeders/ColorSeeder.php:22]: check this [link](https://riptutorial.com/laravel/example/17358/creating-a-seeder)

**Comment by lanorrron** [comment]
[Create seeder para colores](https://trello.com/c/c9WleD6k/2-create-seeder-para-colores)

File: database/seeders/ColorSeeder.php (hunk 1 above)
Comment (lanorrron) [reply database/seeders/ColorSeeder.php:22]: I understand, but what is the difference between  DB and model?

Comment (ttimbal) [reply database/seeders/ColorSeeder.php:22]: DB is for generic tables, and if you have a model for specific table, use that model, the was created fot that

Comment (lanorrron) [reply database/seeders/ColorSeeder.php]: Done

**Review by ttimbal — APPROVED** [review]

**Pull request merged**